### DISCUSSION TITLE
latent-calendar/examples/generation-process/

### DISCUSSION THREAD

**Discussion by giscus** [williambdean/latent-calendar#22] [General]
# latent-calendar/examples/generation-process/

https://wd60622.github.io/latent-calendar/examples/generation-process/

**Comment by galenseilis** [comment]
This could be handy for simulating data for discrete event simulations (e.g. queueing networks) where a detailed breakdown of conditional distribution arrivals is desired. I look forward to when this functionality is available.

**Reply by williambdean** [reply]
I do have some generation logic available in the `latent_calendar.generate` module. Check out the docs [here](https://wd60622.github.io/latent-calendar/modules/generate/). I'm just missing some of the examples at the moment!
